samples/sid_dut/pytest/test_functional.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+from twister_harness.twister_harness_config import DeviceConfig, TwisterHarnessConfig
+from twister_harness.device.factory import DeviceFactory
+from twister_harness.device.device_abstract import DeviceAbstract
+from typing import Generator, Type
+import time
+import logging
+import random
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+load_dotenv(dotenv_path=Path(__file__).parent.joinpath('.env.secrets'))
+load_dotenv(dotenv_path=Path(__file__).parent.joinpath('.env.constants'))
+
+MFG_HEX_PATH = os.getenv('MFG_HEX_PATH')
+
+FAST_RESPONSE_TIMEOUT = int(os.getenv('FAST_RESPONSE_TIMEOUT'))
+FLASH_MFG_TIMEOUT = int(os.getenv('FLASH_MFG_TIMEOUT'))
+DEFAULT_TIMEOUT = int(os.getenv('DEFAULT_TIMEOUT'))
```

Comment:
Why to get those values as env variables? Couldn't it be defined here? Or just keept in one config file (.json or .yml)